src/web/mod.rs

```diff
@@ -323,8 +323,8 @@ mod assets {
     #[derive(Serialize)]
     #[serde(tag = "t", content = "c")]
     pub enum AssetValueDTO {
-        Quantity(u32),
-        BigQuantity(u128),
+        Quantity(String),
+        BigQuantity(String),
```

Comment:
There is a helper type - `StringOf<T>`. It is used for better semantics and type-safety. Please use it here as well.

Also, there is no reason to stringify `Quantity` because `u32` fits into JS's `number` without precision lose.